cluster/gce/configure.sh

```diff
@@ -133,14 +133,14 @@ else
     # Download and untar the release tar ball.
     curl -f --ipv4 -Lo "${TARBALL}" --connect-timeout 20 --max-time 300 --retry 6 --retry-delay 10 "${TARBALL_GCS_PATH}"
     tar xvf "${TARBALL}"
+    # Use crictl installed by kube-up.sh.
+    # TODO(random-liu): Create release tarball for GCE, and do not package crictl.
+    rm "${CONTAINERD_HOME}/usr/local/bin/crictl"
+    rm "${CONTAINERD_HOME}/etc/crictl.yaml"
```

Comment:
We good here? Just checking:
1) will GCE health monitor look for containerd not docker? 
2) How do we ensure now, that the right version of crictl has been installed for a particular install of containerd/cri? Maybe a sanity check here, and report if the version is out of date?

Reply:
`crictl` will ship with Kubernetes on GCE now kubernetes/kubernetes#63357. It has to work with that version of Kubernetes.

If `crictl` works with that version of Kubernetes, but not `containerd`, it means we are using a wrong `containerd` version.